tests/conftest.py

```diff
@@ -396,3 +397,98 @@ def log_cli_level(pytestconfig):
 
     """
     return pytestconfig.getini('log_cli_level') or 'DEBUG'
+
+
+@pytest.fixture(scope="class")
+def run_io_in_background(request):
+    """
+    Run IO during the test execution
+    """
+    if config.RUN['cli_params'].get('io_in_bg'):
```

Comment:
Can we add a log statement saying we will be running IO in the background, just for confirmation from logs that we entered this block of code.

Reply:
sure, will add